client/components/AIAssistantPanel.tsx

```
@@ -353,13 +392,19 @@ export function AIAssistantPanel({
-                {selectedTask === "general" && "💬 Response"}
-              </Text>
+              <View style={styles.resultTitleRow}>
+                {resultMeta ? (
+                  <AppIcon
+                    name={resultMeta.icon}
+                    size={18}
+                    color={theme.primary}
+                    testID={`ai-assistant-result-icon-${selectedTask}`}
+                  />
+                ) : null}
+                <Text style={[styles.resultTitle, { color: theme.textDefault }]}>
```

ESLint

[error] Replace `·style={[styles.resultTitle,·{·color:·theme.textDefault·}]}` with `⏎··················style={[styles.resultTitle,·{·color:·theme.textDefault·}]}⏎················`

(prettier/prettier)